libs/prometheus-operator/config.jsonnet

```diff
@@ -20,8 +24,10 @@ config.new(
         '%s/monitoring.coreos.com_alertmanagers.yaml' % url,
         '%s/monitoring.coreos.com_podmonitors.yaml' % url,
         '%s/monitoring.coreos.com_probes.yaml' % url,
+        '%s/monitoring.coreos.com_prometheusagents.yaml' % url,
```

Comment:
Maybe add a comment here stating this is valid starting at version X, so whenever the 404 behavior changes, there is a trace!

Reply:
Great idea, added in the latest commit 👍